charts_common/lib/src/chart/cartesian/axis/spec/date_time_axis_spec.dart

```diff
@@ -88,11 +88,7 @@ class DateTimeAxisSpec extends AxisSpec<DateTime> {
       other is DateTimeAxisSpec && viewport == other.viewport && super == other;
 
   @override
-  int get hashCode {
-    var hashcode = super.hashCode;
-    hashcode = (hashcode * 37) + viewport.hashCode;
-    return hashcode;
-  }
+  int get hashCode => (super.hashCode * 37) + viewport.hashCode;
```

Comment:
These should all get replaced by something like Object.hash(super.hashCode, viewport.hashCode) rather than this magic "multiply by 37", but that's a later change.